smart-docs/docs/tutorial-download.rst

```diff
@@ -27,14 +33,19 @@ This button is available in one of two places.
 * The bottom of the :ref:`details`:
 
 |annotate-downloadmodel-nodatabutton|
-OR
+OR (if you have labeled data but no model)
 |annotate-downloadmodel-databutton|
-OR
+OR (if you have the same project above with a database connection set up)
```

Comment:
These lines are hard to read in the HTML as they're pretty close together - perhaps add a new line following each image?

Reply:
Agreed